src/components/settings/SettingsLMSTab/ConfigError.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AlertModal, ActionRow, Button } from '@edx/paragon';
+
+const cardText = 'Something went wrong. Please make sure all fields are correctly filled out.';
+
+const ConfigError = ({ isOpen, close }) => (
+  <AlertModal
+    title="Error"
+    isOpen={isOpen}
+    onClose={close}
```

Comment:
should we rename this prop to `onClose` to be consistent with the naming?

Reply:
I just used the variable names consistent from Paragon `const [isOpen, open, close] = useToggle(false);`